<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevaR</class>
 <widget class="QWidget" name="NuevaR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Nueva ruta</string>
    </attribute>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>331</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese la ciudad de llegada: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el pais de salida: </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodCS">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>110</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodPS">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>70</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodCL">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>190</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btCancelarTiempo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el pais de llegada: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese la ciudad de salida: </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodPL">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>150</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btCancelarPrecio_5">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>400</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Crear</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>171</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>NUEVA RUTA</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Nueva ruta menor precio</string>
    </attribute>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>331</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>NUEVA RUTA MENOR PRECIO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btCancelarTiempo_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btCancelarPrecio_6">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>400</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Crear</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodCL_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>210</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodCS_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>130</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodPS_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese la ciudad de salida: </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodPL_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>170</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el pais de llegada: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el pais de salida: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>331</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese la ciudad de llegada: </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Nueva ruta menor tiempo</string>
    </attribute>
    <widget class="QPushButton" name="btCancelarPrecio_7">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>400</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Crear</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>341</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>NUEVA RUTA MENOR TIEMPO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btCancelarTiempo_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodPS_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>80</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el pais de salida: </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodCL_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>200</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_45">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese el pais de llegada: </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodPL_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>160</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtCodCS_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>120</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_46">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>270</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese la ciudad de salida: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_47">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>331</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese la ciudad de llegada: </string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
